client/app/templates/components/project-milestone.hbs

```diff
@@ -27,31 +30,31 @@
     {{/if}}
   </h4>
 
-  {{#if milestone.displayDate}}
-    {{#if (in-array (array "In Progress" "Completed") milestone.statuscode)}}
-      <p class="milestone-dates" data-test-milestone-dates="{{milestone.id}}">
-        
-        {{#if (and milestone.displayDate2 (is-same milestone.displayDate milestone.displayDate2 precision="year"))}}
-          <DateDisplay
-            @date={{milestone.displayDate}}
-            @outputFormat="MMMM D"
-          />
-        {{else}}
-          <DateDisplay
-            @date={{milestone.displayDate}}
-            @outputFormat="MMMM D, YYYY"
-          />
-        {{/if}}
+  {{#if (and
+    (in-array (array "In Progress" "Completed") milestone.statuscode)
+    milestone.displayDate
+  )}}
```

Comment:
Consolidating these two conditions. The date, outcome, link, hearing results, etc. should never show until the milestone is "In progress" or "completed" and there is a recorded date.